docs/src/userguide/special_methods.rst

```diff
@@ -194,6 +194,25 @@ Depending on the application, one way or the other may be better:
 
   These constants can be cimported from the ``cpython.object`` module.
 
+* Use a ``cython.total_ordering`` decorator, which is a re-implementation of
+  the `functools.total_ordering
+  <https://docs.python.org/3/library/functools.html#functools.total_ordering>`_
+  decorator. It can only be used on a ``cdef`` class:
```

Comment:
It might be good to specify that for a Python class, you can just use the original decorator.

Reply:
I'll add a note.